packages/cardpay-sdk/sdk/network-config-utils.ts

```diff
@@ -1,4 +1,4 @@
-import { supportedChains, Networks, networks } from './constants';
+import { supportedChains, Networks, networks, supportedChainsArray } from './constants';
 import { HubConfigResponse } from './hub-config';
 
 type Networkish = string | number | Networks;
```

Comment:
Maybe outside the scope of this PR, but I find it weird that this type is plural.

Reply:
Agreed, I'll add a ticket to rename that